lib/backpex/field.ex

```diff
@@ -1,9 +1,109 @@
 # credo:disable-for-this-file Credo.Check.Refactor.CyclomaticComplexity
 defmodule Backpex.Field do
-  @moduledoc ~S'''
+  @config_schema [
```

Comment:
`default` (function of arity 1) is missing. See https://github.com/naymspace/backpex/blob/0.8.2/lib/backpex/live_resource.ex#L546

Also, can we add `debounce` and `throttle` to the global field options?